spec/validations/manifest_spec.rb

```diff
@@ -692,4 +692,88 @@ def create_package(parameter_hash)
       expect(package).to have_error 'oauth parameter cannot be set to be secure.'
     end
   end
+
+  context 'url is not HTTP or HTTPS' do
+    before do
+      @manifest_hash = {
+        'termsConditionsURL' => 'javascript:alert("terms_conditions_url")',
+        'author' => { 'url' => 'javascript:alert("author_url")' }
+      }
+    end
+
+    it 'terms_conditions_url should have an error' do
+      expect(@package).to have_error("terms_conditions_url must be a valid URL, got \"javascript:alert(\"terms_conditions_url\")\".")
```

Comment:
nit: we can avoid escaping `"` by using single quotes. e.g. `'terms_conditions_url must be a valid URL, got "javascript:alert("terms_conditions_url")".'`

Reply:
done

Reply:
thanks so much for being looking into detail